hack/install.sh

```diff
@@ -17,8 +17,9 @@
 set -e
 
 echo "> Install"
+SOURCE_REPOSITORY=github.com/gardener/gardener
 
-LD_FLAGS="${LD_FLAGS:-$($(dirname $0)/get-build-ld-flags.sh)}"
+LD_FLAGS="${LD_FLAGS:-$($(dirname $0)/get-build-ld-flags.sh "$SOURCE_REPOSITORY")}"
```

Comment:
Hm, maybe instead of passing the flag here, can you default it to `github.com/gardener/gardener` in the `get-build-ld-flags.sh` directly? Then you don't have to touch every file that uses it here.